core/src/main/resources/lib/hudson/queue.jelly

```diff
@@ -103,7 +103,7 @@ THE SOFTWARE.
   </l:pane>
   <!-- schedule updates only for the full page reload -->
   <j:if test="${ajax==null and h.hasPermission(app.READ)}">
-    <div class="widget-refresh-reference" data-id="buildQueue" data-url="${rootURL}/${h.hasView(it,'ajaxBuildQueue')?it.url:''}ajaxBuildQueue"/>
```

Comment:
Breaking change, I need to assess how I can retain compatibility

Reply:
Boilerplate view like https://github.com/jenkinsci/cloudbees-folder-plugin/blob/a514313307249adcf9ee4f197af2d4f94dbd06f5/src/main/resources/com/cloudbees/hudson/plugins/folder/computed/FolderComputation/ajaxBuildQueue.jelly is now ignored.

Reply:
More occurrences
https://github.com/search?q=org%3Ajenkinsci+ajaxBuildQueue&type=code
* simple-queue-plugin
* computer-queue-plugin
* maven-plugin
* ivy-plugin

Reply:
Looked at the two first, doesn't seem to be a problem (they have their own widgets with their own views)